requirements/core-requirements.txt

```diff
@@ -18,3 +18,6 @@ markdown<4,>=3.3
 Jinja2<4,>=2.11; platform_system != 'Windows'
 Jinja2<4,>=3.0; platform_system == 'Windows'
 matplotlib<4
+fastapi<2,>=0.89.1
+uvicorn<2,>=0.20.0
+watchdog<5,>=3.0.0
```

Comment:
Filed https://github.com/mlflow/mlflow/pull/8619. Can we update this PR once it's merged?

Reply:
removing all of these entries